keps/sig-release/create-conformance-doc.md

```diff
@@ -0,0 +1,84 @@
+---
+kep-number: draft-20190121
+title: Provide Conformance Document as part of the release docs for each Kubernetes minor release
+authors:
+  - "@brahmaroutu"
+owning-sig: sig-testing
+participating-sigs:
+  - sig-release
+  - sig-docs
+  - wg-conformance
+reviewers:
+  - @spiffxp
+  - @timothysc
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-01-21
+last-updated: 2019-01-21
+status: provisional
+---
+
+# Provide Conformance Document as part of the release docs for each Kubernetes minor release
+
+## Table of Contents
+
+- [Conformance Document as part of the release docs](#breaking-apart-the-kubernetes-test-tarball)
+  - [Table of Contents](#table-of-contents)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Graduation Criteria](#graduation-criteria)
+  - [References](#references)
+  - [Implementation History](#implementation-history)
+
+## Summary
+
+The Kubernetes release artifacts must include conformance document as part of release documents.
+
+This KEP proposes to add docs/conformance folder with a single conformance document that will 
+contain list of all conformance tests that are part of this release.
+file: docs/conformance/conformance.md 
```

Comment:
bikeshed: I don't like the dupe of conformance, what about "docs/conformance-tests.md" or "docs/conformance-test-list.md"

Reply:
fwiw I would almost prefer it to be a YAML & .md 

Folks have built tooling around this, and a .YAML file is machine readable.

Reply:
Currently as part of release build I have it like this.
srinib@KubeStorage:~/go/src/k8s.io/kubernetes$ **tar -xvf _output/release-tars/kubernetes.tar.gz |grep conformance**
kubernetes/docs/conformance/
kubernetes/docs/conformance/conformance.md

Currently we do not have code to generate yaml. I can open a issue for that.

Reply:
If I'm going to consume this it is most likely on the first time I get certified (where I want to look at all the tests) and then I want to be able to easily `diff` the doc against other versions to see what is changing when I update.